src/fpm_compiler.f90

```diff
@@ -177,6 +179,9 @@ module fpm_compiler
 character(*), parameter :: &
     flag_lfortran_opt = " --fast"
 
+character(*), parameter :: &
+    flag_cpp_preprocessor = " -cpp"
```

Comment:
Do all compilers support the flag `-cpp` for the integrated preprocessor?
I know that Intel Fortran compiler uses `-fpp` (but may support `-cpp` too). Therefore it might be good to add an option to modify this flag.

Reply:
Thanks for pointing it out. I think while writing code, I assumed all compilers support `-cpp` for preprocessing.
Do I need to consider these compilers mentioned [here](https://github.com/fortran-lang/fpm/blob/main/src/fpm_compiler.f90#L5) ?